libs/pages/elewa/social-impact/src/lib/pages/social-impact-page/social-impact-page.component.html

```diff
@@ -1 +1 @@
-<p>social-impact-page works!</p>
+<elewa-group-elewa-group-image-and-text-banner></elewa-group-elewa-group-image-and-text-banner>
```

Comment:
Create a component for the hero section and call that here then pass the required values